fairroot/base/steer/FairRunAna.cxx

```diff
@@ -614,6 +585,25 @@ void FairRunAna::TerminateRun()
 }
 //_____________________________________________________________________________
 
+/**
+ * \sa FairRunOnline::CheckRunIdChanged
+ */
+void FairRunAna::CheckRunIdChanged()
+{
+    auto newrunid = GetEvtHeaderRunId();
+    if (newrunid == fRunId) {
+        return;
+    }
+    LOG(info) << "FairRunAna::CheckRunIdChanged() Detected changed RunID from " << fRunId << " to " << newrunid;
+    fRunId = newrunid;
+    if (fStatic) {
+        LOG(info) << "FairRunAna::CheckRunIdChanged: ReInit not called because initialisation is static.";
```

Comment:
Please change the `info` to `debug`.

From what I remember in certain edge cases this could result in console flooded with ChangeRunId messages.

Reply:
Did so, also in FairRunOnline, where it already was `info` in the old code.

Reply:
Done.